services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -120,6 +120,60 @@ class IntentionalRerouteSpec extends BaseSpecification {
         [thinIsl, islUtils.reverseIsl(thinIsl)].each { database.revertIslBandwidth(it) }
     }
 
+    def "Should be able to reroute to a path with not enough bandwidth available in case ignoreBandwidth: true"() {
+        given: "A flow with alternate paths available"
+        def switches = topology.getActiveSwitches()
+        List<List<PathNode>> allPaths = []
+        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
+                .findAll { src, dst -> src != dst }.unique { it.sort() }.find { Switch src, Switch dst ->
+            allPaths = database.getPaths(src.dpId, dst.dpId)*.path
+            allPaths.size() > 1
+        } ?: assumeTrue("No suiting switches found", false)
+        def flow = flowHelper.randomFlow(srcSwitch, dstSwitch)
+        flow.maximumBandwidth = 10000
+        flow.ignoreBandwidth = true
+        flowHelper.addFlow(flow)
+        def currentPath = PathHelper.convert(northbound.getFlowPath(flow.id))
+
+        when: "Make the current path less preferable than alternatives"
+        def alternativePaths = allPaths.findAll { it != currentPath }
+        alternativePaths.each { pathHelper.makePathMorePreferable(it, currentPath) }
+
+        and: "Make all alternative paths to have not enough bandwidth to handle the flow"
+        def currentIsls = pathHelper.getInvolvedIsls(currentPath)
+        def changedIsls = alternativePaths.collect { altPath ->
+            def thinIsl = pathHelper.getInvolvedIsls(altPath).find {
+                !currentIsls.contains(it) && !currentIsls.contains(islUtils.reverseIsl(it))
+            }
+            def newBw = flow.maximumBandwidth - 1
+            database.updateLinkMaxBandwidth(thinIsl, newBw)
```

Comment:
`[thinIsl, thinIsl.reversed].each { database.updateLinkMaxBandwidth(it, newBw) }`

Reply:
Done.